config/runtime.exs

```diff
@@ -139,17 +139,52 @@ end
 ###################################
 
 if System.get_env("RELEASE") == "central_backend" or config_env() == :dev do
-  {:ok, currency_config_json} =
-    "./apps/game_backend/priv/currencies_rules.json"
-    |> File.read()
-
-  config :game_backend, :currencies_config, Jason.decode!(currency_config_json)
-
   {:ok, daily_rewards_json} =
     Application.app_dir(:game_backend, "priv/daily_rewards_rules.json")
     |> File.read()
 
   config :game_backend, :daily_rewards_config, Jason.decode!(daily_rewards_json)
+
+  arena_prestige_ranks_json =
+    Application.app_dir(:game_backend, "priv/arena_prestige_ranks.json")
+    |> File.read!()
+    |> Jason.decode!()
+
+  arena_prestige_ranks =
+    Enum.reduce(arena_prestige_ranks_json["ranks"], [], fn rank, acc ->
+      Enum.reduce(rank["sub_ranks"], acc, fn sub_rank, acc ->
+        entry = %{
+          rank: rank["rank"],
+          sub_rank: sub_rank["sub_rank"],
+          min: sub_rank["min"],
+          max: sub_rank["max"]
+        }
+
```

Comment:
Just to be sure, all what this is doing is converting the map keys from string to atoms right?

Reply:
Yes, but also a bit more. Its also making the entries more similar to what we would have if this was stored in the DB (every entry with `rank`) without actually duplicating things in the config